config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
+    cost:
+      title: "Reduce overhead"
+      description: "Contributing regularly to open source saves money and developer hours by reducing the custom software you need to maintain."
+    reputation:
+      title: "Attract and retain talent"
+      description: "Employees that work on open source boost their, and their employer's, reputation."
+    plan:
+      title: "Open Source Friday in your organization"
+      description: "Improving your developer culture is too important to be left unplanned."
+      encourage:
+        title: "Set aside time"
+        description: "Each Friday, encourage your managers, employees, and coworkers to spend at least two hours working on the open source projects your organization already uses."
+      policy:
+        title: "Make it a habit"
+        description: "Create clear, internal policies so employees can easily and consistently contribute to open source."
+
+  # app/views/home/maintainers.html.erb
+  maintainers:
+    title: "Build software with help from others"
+    description:  "Every Friday, give your software project some love and make it easier for others to contribute."
+    involve:
+      title: "Help people discover your project"
+      description: "Creating a project is only the beginning. Why not spread the&nbsp;word?"
+      workload:
+        title: "Reduce your workload:"
+        description_html: "Ensure your project's README file details what your project does, its scope and contribution needs, and how to contribute to < a href=\"%{url}\">leverage your community</a>."
+      promote:
+        title: "Promote your work"
+        description_html: "Consider creating a Twitter account, blog posts and website to <a href=\"%{url}\">help people find and follow your project</a>."
+    growing:
+      title: "Grow your community"
+      description: "Everybody was new once. Ease someone else's first-time contribution jitters."
+      document:
+        title: "Document everything"
+        description_html: "<a href=\"%{survey}\">GitHub's 2017 Open Source Survey</a> showed incomplete or confusing documentation is the biggest problem for open source users. Ensure you are <a href=\"%{guide}\">documenting your processes </a>  for how to contribute to your project."
+      help:
+        title: "Get help"
+        Look_out_html: "Look out for promising potential contributors or maintainers to <a href=\"%{url}\"> share ownership of your project.</a> Make an effort to welcome first-time contributors in particular."
+    workflow:
+      title: "Improve your workflow"
+      description: "Learn from other maintainers on how to be more effective."
+      boundaries:
+        title: "Set boundaries"
+        description_html: "Decide on what the scope of the project is and <a href=\"%{pause}\">set clear boundaries for yourself</a>. Saying no <a href=\"%{sayno}\"> is as important as saying yes</a>."
```

Comment:
No we cannot avoid HTML here. Some language has different syntax than English. With this translator can move link to their preferable place in a translated sentence.